lib/KMS.js

```diff
@@ -14,8 +14,12 @@ class KMS {
      * constructor - Build our KMS object
      * @param  string region The AWS region we're working with
      */
-    constructor(region) {
+    constructor(region, keyId) {
```

Comment:
Would be great if we could use an ES6 default param here instead of the check below